roles/mailserver/tasks/dmarc.yml

```diff
@@ -1,5 +1,11 @@
-- name: Install OpenDMARC milter
-  apt: pkg=opendmarc state=installed update_cache=yes
+- name: Install OpenDMARC milter and related packages
+  apt: pkg={{item}} state=installed
+  with_items:
+    - mysql-server
```

Comment:
Installing something like mysql might warrant a whole separate task, and might need some ancillary tasks for, say, configuring it securely.  What do y'all think?